rfq/negotiator.go

```diff
@@ -278,6 +278,16 @@ func (n *Negotiator) HandleIncomingBuyRequest(
 		}
 	}
 
+	// Reject the quote request if a price oracle is unavailable.
+	if n.cfg.PriceOracle == nil {
```

Comment:
Circling back to my previous question, shouldn't it be impossible to not have a price oracle configured? At least as a normal user? You either have to use an RPC implementation or the mock?
I think even if the end-user is running with the `--skip...` flag they should be using the mock oracle (it just isn't actually queried, bit it's always hooked up internally).

Reply:
I think it should be valid for a user to start tapd without specifying a price oracle address (not even the mock string). A user might not care about lightning channel support.

Roas suggests here https://github.com/lightninglabs/taproot-assets/pull/915#discussion_r1615539072
that if the price oracle isn't specified we just reject all incoming quotes.

Reply:
Makes sense :+1: